sdk/communication/Azure.Communication.Sms/tests/SmsClientTestEnvironment.cs

```diff
@@ -7,8 +7,11 @@ namespace Azure.Communication.Sms.Tests
 {
     public class SmsClientTestEnvironment : CommunicationTestEnvironment
     {
-        internal const string PhoneNumberEnvironmentVariableName = "AZURE_PHONE_NUMBER";
+        internal const string ToPhoneNumberEnvironmentVariableName = "AZURE_PHONE_NUMBER";
```

Comment:
+1

Reply:
@RezaJooyandeh  what do you mean here? I added the To and From PhoneNumberEnvironmentVariableName even though they map to the same environment variable in order for the tests to read better. If this isn't necessary I can revert :)

Reply:
Oh no sorry. I just wanted to say I like this.